lib/ui/home/home_screen.dart

```diff
@@ -56,10 +57,7 @@ class HomeScreenState extends ConsumerState<HomeScreen> {
     final uiModels = ref.watch(_surveysStreamProvider).value ?? [];
     return ref.watch(homeViewModelProvider).when(
           init: () => const Center(
-            // TODO: replace CircularProgressIndicator with the shimmer effect.
-            child: CircularProgressIndicator(
-              valueColor: AlwaysStoppedAnimation(Colors.white),
-            ),
+            child: SkeletonLoadingScreen(),
```

Comment:
@AVI5HEK I think we can remove `const Center` and use `const SkeletonLoadingScreen()` instead

Reply:
@Wadeewee done in https://github.com/nimblehq/ic-flutter-avishek/pull/46/commits/750980c926f3cbbe217abcb70c953733a3e6dfb0